deps/rabbit/src/rabbit_msg_store.erl

```diff
@@ -711,8 +1007,9 @@ init([VHost, Type, BaseDir, ClientRefs, StartupFunState]) ->
     process_flag(trap_exit, true),
     pg:join({?MODULE, VHost, Type}, self()),
 
-    ok = file_handle_cache:register_callback(?MODULE, set_maximum_since_use,
-                                             [self()]),
+    %% @todo Remove. DONENOW
+%    ok = file_handle_cache:register_callback(?MODULE, set_maximum_since_use,
+%                                             [self()]),
```

Comment:
review, this might still be needed based on the final changes as FHC is used more than I would have wanted